README.md

```diff
@@ -227,10 +227,21 @@ $ npm run component-gen
 
 ## ➤ How to release
 
-As mentioned before, we are using [lerna](https://github.com/lerna/lerna), to be able to publish you need enable your [two-factor authentication in npm](https://docs.npmjs.com/configuring-two-factor-authentication)
+We are using [lerna to publish](https://github.com/lerna/lerna/tree/master/commands/publish#readme) our elements
+
+- Create a branch for releasing。 `git branch release`
+- `git push release` and set the upstream.
+- `lerna publish`
+- If you have problems while releasing. The following may help
 
 ```shell
-$ NPM_CONFIG_OTP=123456 lerna publish`   # Push the packages to registry.npmjs.org
+# delete local tag '12345'
+git tag -d 12345
```

Comment:
Can you change 12345 to the `v0.5.0` or something that shows the format of our release tag?